### PR TITLE
Yoruba lang

### PR DESCRIPTION
**Pull request by cletusajibade**
Added Yoruba language po file. @MatteoPE

### CODE REVIEW

File: Angular/src/app/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -33,6 +33,10 @@
           <mat-icon>{{ isCurrentLanguage('it') ? 'check' : '' }}</mat-icon>
           <span>Italiano (it)</span>
         </button>
+        <button mat-menu-item (click)="changeLanguageTo('yo')">
+          <mat-icon>{{ isCurrentLanguage('it') ? 'check' : '' }}</mat-icon>
```
Comment (pannapat) [inline Angular/src/app/sidebar/sidebar.component.html]: This should be 'yo'

File: Angular/src/assets/i18n/yo.po [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Translation File
+msgid ""
+msgstr ""
+
+
+msgid "Akojopo Èdè"
+msgstr ""
```
Comment (pannapat) [inline Angular/src/assets/i18n/yo.po]: We need to put the translation to the 'msgstr' not 'msgid'

**Review by pannapat — CHANGES_REQUESTED** [review]

File: Angular/src/app/sidebar/sidebar.component.html (hunk 1 above)
Comment (cletusajibade) [reply Angular/src/app/sidebar/sidebar.component.html]: done

File: Angular/src/assets/i18n/yo.po (hunk 2 above)
Comment (cletusajibade) [reply Angular/src/assets/i18n/yo.po]: updated.

**Review by pannapat — APPROVED** [review]
Great! It works now :)

**Pull request merged**